### DISCUSSION TITLE
BMP预合成导出的内容还可以替换内容吗?

### DISCUSSION THREAD

**Discussion by cgeffect** [Tencent/libpag#1882] [Q&A] [answered]
我有一个图层, 现在我在图层上加了一个PAG不支持的效果, 比如是一个调整图层颜色的效果, 又或者是一个是使图层扭曲的效果, 那么此时导出PAG时设置BMP预合成导出. 导出之后这个图层的内容应该是不可以替换内容的吧, 预合成已经把图层和效果整合在一起导出的序列帧了, 是这样的吗?

在官网中看到 
在纯矢量导出方式上支持更多 AE 特性的同时，还引入了BMP预合成结合矢量的混合导出能力，实现支持所有 AE 特性的同时又能保持动效运行时的可编辑性。

BMP的预合成导出是如何实现可编辑性的?

**Comment by 0x1306a94** [comment]
@cgeffect BMP 后不支持编辑了

**Comment by kevingpqi123** [comment, marked as answer]
所谓BMP预合成结合矢量的混合导出，指的是需要编辑的内容不要选择 BMP 预合成，还是矢量的，其余内容可以选择 BMP 预合成， BMP 预合成本身不支持可编辑
